### ISSUE TITLE
Request for Query Embedding Vectors for Training Set

### ISSUE THREAD

**Issue by weiwch** [microsoft/MS-MARCO-Web-Search#11]
Thank you for your great work. I'd like to evaluate a learnable ANN algorithm on this dataset, but I need the query embedding vectors for the training set. However, you only provided 9,374 embedding vectors for test set. I would like to know how to obtain the query embedding vectors for the training set. Could you please provide the file for the training set, or the method to generate corresponding vectors from text?
